data_dictionary/storage_options.hh

```diff
@@ -29,6 +29,8 @@ struct storage_options {
     value_type value = local{};
 
     storage_options() = default;
+    storage_options(storage_options&&) = default;
+    storage_options(const storage_options&) = default;
```

Comment:
Seems unnecessary, why are those needed? The compiler should synthesize them without help.